### ISSUE TITLE
Internal server error on `Configuration` page, supposedly when there is no data

### ISSUE THREAD

**Issue by saveriomiroddi** [danmayer/coverband#403]
Hello!

I have Coverband setup on an app. I suppose that there is no coverage data available, since the main page doesn't show any data:

![image](https://user-images.githubusercontent.com/1595356/94542556-3b8e9400-0249-11eb-90a3-95e5ef0a7e54.png)

This is OK. However, the problem is that when I click on `Configuration`, I get an `Internal server error`, with the following backtrace:

```
Unexpected error while processing request: undefined method `bytesize' for nil:NilClass
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/coverband-5.0.1/lib/coverband/adapters/redis_store.rb:47:in `size'
  (erb):25:in `format_settings'
  /home/myuser/.rvm/rubies/ruby-myversion/lib/ruby/2.6.0/erb.rb:901:in `eval'
  /home/myuser/.rvm/rubies/ruby-myversion/lib/ruby/2.6.0/erb.rb:901:in `result'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/coverband-5.0.1/lib/coverband/utils/html_formatter.rb:53:in `format_settings'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/coverband-5.0.1/lib/coverband/utils/html_formatter.rb:33:in `format_settings!'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/coverband-5.0.1/lib/coverband/reporters/web.rb:86:in `settings'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/coverband-5.0.1/lib/coverband/reporters/web.rb:56:in `call'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/thin-1.7.2/lib/thin/connection.rb:86:in `block in pre_process'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/thin-1.7.2/lib/thin/connection.rb:84:in `catch'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/thin-1.7.2/lib/thin/connection.rb:84:in `pre_process'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/thin-1.7.2/lib/thin/connection.rb:53:in `process'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/thin-1.7.2/lib/thin/connection.rb:39:in `receive_data'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/eventmachine-1.2.7/lib/eventmachine.rb:195:in `run_machine'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/eventmachine-1.2.7/lib/eventmachine.rb:195:in `run'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/thin-1.7.2/lib/thin/backends/base.rb:73:in `start'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/thin-1.7.2/lib/thin/server.rb:162:in `start'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/rack-2.1.4/lib/rack/handler/thin.rb:24:in `run'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/rack-2.1.4/lib/rack/server.rb:329:in `start'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/rack-2.1.4/lib/rack/server.rb:170:in `start'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/coverband-5.0.1/lib/coverband/utils/tasks.rb:16:in `block (2 levels) in <top (required)>'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/rake-12.3.3/lib/rake/task.rb:273:in `block in execute'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/rake-12.3.3/lib/rake/task.rb:273:in `each'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/rake-12.3.3/lib/rake/task.rb:273:in `execute'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/rake-12.3.3/lib/rake/task.rb:214:in `block in invoke_with_call_chain'
  /home/myuser/.rvm/rubies/ruby-myversion/lib/ruby/2.6.0/monitor.rb:235:in `mon_synchronize'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/rake-12.3.3/lib/rake/task.rb:194:in `invoke_with_call_chain'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/rake-12.3.3/lib/rake/task.rb:183:in `invoke'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/rake-12.3.3/lib/rake/application.rb:160:in `invoke_task'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/rake-12.3.3/lib/rake/application.rb:116:in `block (2 levels) in top_level'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/rake-12.3.3/lib/rake/application.rb:116:in `each'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/rake-12.3.3/lib/rake/application.rb:116:in `block in top_level'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/rake-12.3.3/lib/rake/application.rb:125:in `run_with_threads'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/rake-12.3.3/lib/rake/application.rb:110:in `top_level'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/rake-12.3.3/lib/rake/application.rb:83:in `block in run'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/rake-12.3.3/lib/rake/application.rb:186:in `standard_exception_handling'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/rake-12.3.3/lib/rake/application.rb:80:in `run'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/gems/rake-12.3.3/exe/rake:27:in `<top (required)>'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/bin/rake:23:in `load'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/bin/rake:23:in `<main>'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/bin/ruby_executable_hooks:24:in `eval'
  /home/myuser/.rvm/gems/ruby-myversion@mygemset/bin/ruby_executable_hooks:24:in `<main>'
```

I'm not sure if this is due to the lack of coverage data, however, intuitively, opening the configuration should not yield an error.

**Comment by danmayer** [comment]
OK, yeah... I can fix this... basically since there is no data yet it isn't setup to handle the empty dataset case. I will resolve

**Comment by danmayer** [comment]
fixed in release 5.0.2
